sdk/core/azure-core-http-okhttp/src/test/java/com/azure/core/http/okhttp/OkHttpAsyncHttpClientBuilderTests.java

```diff
@@ -531,18 +531,6 @@ public void buildWithConfigurationProxy(boolean shouldHaveProxy, Configuration c
     private static OkHttpClient okHttpClientWithProxyValidation(boolean shouldHaveProxy, Proxy.Type proxyType) {
         return new OkHttpClient.Builder()
             .eventListener(new TestEventListenerValidator(shouldHaveProxy, proxyType))
-            // Use a custom Dispatcher and ExecutorService which overrides the uncaught exception handler.
```

Comment:
Any reason this code was removed?

Reply:
Same reason, I merge the preview PR here: https://github.com/Azure/azure-sdk-for-java/pull/23123/files#diff-5fc80e8d55aec92ad4626a581a93bd54c10beaa4eb240e8e4c5645bf670da5b9L492